components/raftstore/src/store/fsm/peer.rs

```diff
@@ -918,6 +923,19 @@ where
         }
     }
 
+    fn on_persisted_msg(&mut self, peer_id: u64, ready_number: u64, _send_time: Instant) {
+        if peer_id != self.fsm.peer_id() {
+            error!(
```

Comment:
Can this happen?

Reply:
It should not happen by design because the region should be destroyed after all io task has finished. It's used to defend against some unexpected bugs in my opinion.

Reply:
Then how about panicking?

Reply:
Maybe too strict? It seems no harm to ignore this msg.

Reply:
I was thinking about mismatched messages between regions.